.github/workflows/esp32-build.yaml

```diff
@@ -147,6 +154,7 @@ jobs:
         tar -xf ${QEMU_RISCV32_DIST} -C /opt && rm ${QEMU_RISCV32_DIST}
 
     - name: Install pytest and pytest-embedded plugins
+      if: matrix.esp-idf-target != 'esp32p4'
```

Comment:
Let's add a comment about enabling again esp32p4 when available, and also let's check if we can have any insight about when.

Reply:
Absolutely, I have opened an issue in the esp-idf repo and will add any details about future availability. I suspect this is a long way off. ESP32-S3 QEMU support should be added soon, but these chips have been available for about 5 years now.

Reply:
A comment was added at the first occurrence and todos added to each occurrence to remove these conditional exclusions when ESP32P4 support is added to `espressif/qemu`. There is no expected time line yet. An inquiry was also posted on the esp32.com forum: https://esp32.com/viewtopic.php?t=45665. When an expected release date, or esp-idf version with compatibility is known we can update the comment to make those changes after a specific release.